home/names.asm

```diff
@@ -7,7 +7,7 @@ NamesPointers:: ; 33ab
 	dbw 0, PartyMonOT
 	dbw 0, OTPartyMonOT
 	dba TrainerClassNames
-	dbw $4, $4b52 ; within PackMenuGFX
+	dbw $4, PackMenuGFX + 60
```

Comment:
This comment was just to indicate that it wasn't a valid address. It's not actually referencing PackMenuGFX. My guess would be it's a pointer to MoveDescriptions (0b:4b52). The `$4, $4b52` is `$4, $4000` in Gold. The address of MoveDescriptions in Gold is also at 0x4000 (6d:4000). The bank is wrong still, but 04:4000 in Gold was GetPlayerMovement, so the bank was probably wrong to begin with.